cli/src/complete.rs

```diff
@@ -218,18 +218,58 @@ fn revisions(revisions: &str) -> Vec<CompletionCandidate> {
             .arg("--revisions")
             .arg(revisions)
             .arg("--template")
-            .arg(r#"change_id.shortest() ++ " " ++ if(description, description.first_line(), "(no description set)") ++ "\n""#)
+            .arg(
+                r#"
+                change_id.shortest() ++ "\t" ++
+                bookmarks.map(|b| separate("@", b.name(), b.remote())).join(",") ++ "\t" ++
+                tags.map(|t| t.name()).join(",") ++ " " ++
+                if(description, description.first_line(), "(no description set)") ++ "\n"
+                "#,
+            )
```

Comment:
Should we omit old tags and branches? I don't think it's uncommon to query changes between some old tagged revision (specified by tag) and recent commits (specified by `@` or change id.)

Reply:
That's true. The revisions are limited to 100, any bookmarks / tags not pointing to this limited list are not shown. I'll have to shell out twice to improve that.

Reply:
Yeah, we'll need a helper struct.

BTW, for "all_revisions" query, it might make sense to use `revsets.short-prefixes` (instead of `--limit 100`) to collect change ids which the user would be interested in. For "mutable_revisions", using `mutable()` seems good.